docs/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,288 @@
+    |Comment `Duplicate of #<issue number>` | YES +----------|----------+                                                        
+    |Remove needs-triage label              -------  Duplicate issue?   |                                                        
+    |label: triage/duplicate                |     |                     |                                                        
+    +-----|---------------------------------+     +-----------|---------+                                                        
+          |                                                   |NO                                                                
+          |  +-------------------------+  NO   +-----------------------------+                                                   
+          |  |Add comment              |--------  Is this a valid feature?   |                                                   
+          |  |Remove needs-triage label|       |                             |                                                   
+          |  +------|------------------+       +--------------|--------------+                                                   
+          |         |                                         | YES                                                              
+          |         |                                         |                                                                  
+          |         |                         +---------------|--------------+                                                                                                    
+          |         |                         | label: type/feature          |                                                   
+        +-|---------|---+       +--------+    | Remove needs-triage label    |                                                   
+        |  Close issue  |       |  Done  ------ Remove type/feature-request  |                                                   
+        |               |       |        |    | milestone?                   |                                                   
+        +---------------+       +--------+    +------------------------------+           
+```
+If the author does not respond to a request for more information within the timespan of a week, close the issue with a kind note stating that the author can request for the issue to be reopened when the necessary information is provided.
+
+In some cases you may receive a request you do not wish to accept.  Perhaps the request doesn't align with the project scope or vision.  It is important to tactfully handle contributions that don't meet the project standards.
+
+1. Acknowledge the person behind the contribution and thank them for their interest and contribution
+2. Explain why it didn't fit into the scope of the project or vision
+3. Don't leave an unwanted contributions open.  Immediately close the contribution you do not wish to accept
```

Comment:
Change "contribution" to "request" in "Immediately close the contribution ..."

Reply:
Updated as suggested

Reply:
There was another instance of the same thing in the referenced line. Please make the same change for that instance and other similar instances in other lines. See the new comment for this below specifying the current line numbers that need this change.